extensions/resource-deployment/src/utils.ts

```diff
@@ -17,16 +17,20 @@ export function getDateTimeString(): string {
 }
 
 export function setEnvironmentVariablesForInstallPaths(tools: ITool[]): void {
-	let installationPaths: Set<string> = new Set<string>();
+	let installationPaths: string[] = [];
 	tools.forEach(t => {
-		// construct an env variable name with NoteBookEnvironmentVariablePrefix prefix
-		// and tool.name as suffix, making sure of using all uppercase characters and only _ as separator
-		const envVarName: string = `${NoteBookEnvironmentVariablePrefix}${t.name.toUpperCase().replace(/ |-/, '_')}`;
-		process.env[envVarName] = t.installationPath;
-		installationPaths.add(path.resolve(path.dirname(t.installationPath)));
-		console.log(`setting env var:'${envVarName}' to: '${t.installationPath}'`);
+		if (t.installationPath) {
+			// construct an env variable name with NoteBookEnvironmentVariablePrefix prefix
+			// and tool.name as suffix, making sure of using all uppercase characters and only _ as separator
+			const envVarName: string = `${NoteBookEnvironmentVariablePrefix}${t.name.toUpperCase().replace(/ |-/, '_')}`;
```

Comment:
What is this regex supposed to be replacing? The | doesn't seem correct and this isn't using a global search so it'll only replace the first instance

Reply:
Oops, missed the space before the |. So that part looks fine. But you should still be doing a global replace.